fast/stages/03-gke-multitenant/dev/variables.tf

```diff
@@ -241,6 +241,7 @@ variable "nodepools" {
       node_taints       = list(string)
     })
     preemptible = bool
```

Comment:
Does it make sense to support both preemptible and spot, or can we just keep spot?